scripts/cleanup.py

```diff
@@ -200,7 +201,33 @@ def main():
         help="Root path of rageshakes (eg /home/rageshakes/bugs/)",
     )
 
+    parser.add_argument(
+        "--repeat-delay-hours",
+        dest="repeat_delay_hours",
+        type=float,
+        help="After each execution, wait this number of hours then run again. An alternative to configuring a cronjob for ongoing cleanup.",
```

Comment:
Possibly clearer, ymmv

```suggestion
        help="""
            Instead of exiting after execution, run repeatedly, waiting this number of hours between each execution.
            An alternative to configuring a cronjob for ongoing cleanup.
            """,
```